microSALT/utils/referencer/utils.py

```diff
@@ -0,0 +1,38 @@
+from microSALT.utils.referencer.constants import (
+    Escherichia,
+    Klebsiella,
+    ESCHERICHIA_REFERENCE,
+    KLEBSIELLA_REFERENCE,
+)
+
+
+def get_species(organism_name: str) -> str:
+    """Returns the species of the organism name."""
+    return organism_name.lower().split()[-1]
+
+
+def is_escherichia(species_name: str) -> bool:
+    """Checks if the organism name is a proper Escherichia."""
+    return any(species_name == member.value for member in Escherichia)
+
+
+def is_klebsiella(species_name: str) -> bool:
+    """Checks if the organism name is a proper Klebsiella."""
+    return any(species_name == member.value for member in Klebsiella)
+
+
+def get_reference_if_enterobacteriaceae(organism_name: str) -> str:
+    """Returns the reference for the organism name if it belongs to Enterobacteriaceae."""
+    species: str = get_species(organism_name)
+    GENUS_REFERENCE_MAP = {
```

Comment:
Why is a dict (python lingo for maps) used here instead of just a list, as the dict keys are never used? It seems to me a list with the tuples would make the code simpler and clearer.

I also think Python dicts are not guaranteed to be sorted (although they often are in practice), which might or might not be a problem here?

Reply:
I meant to make it so that we get the correct tuple by the genus name, I missed that so nice catch! Now I will only try to check for it if it is the correct genus, and then check if the species are among the expected ones.